frontends/web/src/routes/account/send/send.tsx

```diff
@@ -467,11 +463,7 @@ class Send extends Component<Props, State> {
     private toggleCoinControl = () => {
         this.setState(({ activeCoinControl }) => {
             if (activeCoinControl && this.utxos.current) {
```

Comment:
this branch looks like dead code. toggleCoinControl() is only ever called when activeCoinControl is false, from send.tsx. 

Maybe its a relict from a past where the button was actually a toggler, not a dialog opener.

I propose to rename this function to 'openCoinControl` or similar, and to rename 'Toggle coin control' to 'Open coin control' or similar. cc @thisconnect wdyt?

Reply:
ack, makes sense.